provider/postgis/postgis.go

```diff
@@ -315,45 +316,38 @@ func BuildDBConfig(opts *DBConfigOptions) (*pgxpool.Config, error) {
 		return nil, err
 	}
 
-	dbconfig.ConnConfig.LogLevel = pgx.LogLevelWarn
 	dbconfig.ConnConfig.RuntimeParams = opts.GetRuntimeParams()
-	var hstore hstoreOID
+
+	// NOTE: reflects previous pgx/v4 behaviour of passing pgx.loglevelwarn
+	logAdapter := NewLoggerAdapter()
+	tracer := &tracelog.TraceLog{
+		Logger:   logAdapter,
+		LogLevel: tracelog.LogLevelWarn,
+	}
+	dbconfig.ConnConfig.Tracer = tracer
+
+	var hstoreOID uint32
 
 	dbconfig.AfterConnect = func(ctx context.Context, conn *pgx.Conn) error {
-		// The AfterConnect call runs everytime a new connection is acquired,
-		// including everytime the connection pool expands. The hstore OID
-		// is not constant, so we lookup the OID once per provider and store it.
-		// Extensions have to be registered for every new connection.
-
```

Comment:
Has this behavior changed? Is it okay to re scan the `hstoreOID` on every connect now?

Reply:
Good catch. Must have slipped through during testing. While it's not impacting functionality to leave it like this, we can keep the previous behaviour and reduce overhead of requesting the oid on every connection entering the pool.